index.html

```diff
@@ -2512,7 +2483,7 @@ <h3 class="wow fadeInDown">Silver Sponsors</h3>
       <ul class="tg-sponsors col-md-4">
         <li>
           <figure class="tg-sponder">
-            <a data-toggle="modal" href="#zeomega-modal" target="_blank" href="http://www.zeomega.com/">
+            <a href="http://www.zeomega.com/" target="_blank">
```

Comment:
This is supposed to open up a modal. In the modal, you can link their website which can open in another tab.

Reply:
So, rever the code where you've removed the modal code :) and do the changes for the link in the modal itself.